examples/with-reason-react/src/client.js

```diff
@@ -3,7 +3,7 @@ import './client.css';
 import React from 'react';
 import { render } from 'react-dom';
 
-const App = require('../lib/js/src/app').comp; // BuckleScript output directory
+import App from '../lib/js/src/App'; // BuckleScript output directory
```

Comment:
We can change "in-source" in bsconfig.json to true and just require it like normal, `import App from './App'`